<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_Main</class>
 <widget class="QDialog" name="Dialog_Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>596</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive/>
    <disabled/>
   </palette>
  </property>
  <property name="windowTitle">
   <string>YouTube Downloader</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="Label_Links">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Links</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_Paste">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>70</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Paste 
from 
Clipboard</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_Video_Down">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>340</y>
     <width>161</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Video Download</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_Audio_Down">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>340</y>
     <width>181</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Audio Download</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_List_File">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>420</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>List File</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEdit_ListFile_Path">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>440</y>
     <width>580</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_Select_File">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>430</y>
     <width>100</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_Close_App">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>500</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Close App</string>
   </property>
  </widget>
  <widget class="QListWidget" name="ListWidget_Links">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>581</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEdit_Link">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>294</y>
     <width>581</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_Add">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>280</y>
     <width>100</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_Delete">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>220</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="TextBrowser_Status">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>490</y>
     <width>501</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Btn_Paste</sender>
   <signal>clicked()</signal>
   <receiver>ListWidget_Links</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>670</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Btn_Add</sender>
   <signal>clicked()</signal>
   <receiver>ListWidget_Links</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>669</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Btn_Delete</sender>
   <signal>clicked()</signal>
   <receiver>ListWidget_Links</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>669</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LineEdit_Link</sender>
   <signal>returnPressed()</signal>
   <receiver>ListWidget_Links</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Btn_Select_File</sender>
   <signal>clicked()</signal>
   <receiver>LineEdit_ListFile_Path</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>669</x>
     <y>459</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>454</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
